x-pack/plugins/actions/server/feature.ts

```diff
@@ -28,15 +28,15 @@ export const ACTIONS_FEATURE = {
   app: [],
   order: FEATURE_ORDER,
   management: {
-    insightsAndAlerting: ['triggersActions'],
```

Comment:
This constant is in the `triggers_actions_ui` plugin, not the `actions` plugin, and attempting to import it causes an unresolvable `no_boundary_crossing` error. Because `triggersActions` is already here as a literal and not as an imported constant I think we should just leave it as-is.